package.json

```diff
@@ -437,6 +446,11 @@
                     "command": "vscode-edge-devtools-view.refresh",
                     "when": "view == vscode-edge-devtools-view.targets",
                     "group": "navigation"
+                },
+                {

```

Comment:
what is the difference between these 2 commands? the registration that has a title and icon and the second one that has a when and a group?

Reply:
The first section with the title and icon is the actual registration of the command whereas the second one is to place the existing command to the extension title bar like this:
![image](https://user-images.githubusercontent.com/14304598/110155366-5c2fbc80-7d9a-11eb-8201-dce8f617fbd4.png)